src/app/core/classes/hardware/vdev-labels-svg.ts

```diff
@@ -269,33 +266,33 @@ export class VDevLabelsSVG {
     });
   }
 
-  showTile(devname: string) {
-    const targetEl = this.getParent().querySelector('rect.tile_' + devname);
+  showTile(devname: string): void {
+    const targetEl: HTMLElement = this.getParent().querySelector('rect.tile_' + devname);
     if (targetEl) {
-      targetEl.style.opacity = 1;
+      targetEl.style.opacity = '1';
     }
   }
 
-  hideTile(devname: string) {
-    const targetEl = this.getParent().querySelector('rect.tile_' + devname);
+  hideTile(devname: string): void {
+    const targetEl: HTMLElement = this.getParent().querySelector('rect.tile_' + devname);
     if (targetEl) {
-      targetEl.style.opacity = 0;
+      targetEl.style.opacity = '0';
     }
   }
 
-  hideAllTiles(tiles: any[], exceptions?: string[]) {
+  hideAllTiles(tiles: NodeListOf<any>, exceptions?: string[]): void {
     tiles.forEach((item) => {
       item.style.opacity = 0;
```

Comment:
why opacity value has string type in `hideTile()` and number in `hideAllTiles()`?

Reply:
Because it's incorrect in `hideAllTitles` because NodeListOf<any> is not fully typed :)